cypress/e2e/5-ndv.cy.ts

```diff
@@ -163,5 +163,70 @@ describe('NDV', () => {
 			ndv.getters.outputPanel().find('[class*=_pagination]').should('not.exist');
 			ndv.getters.outputPanel().find('[data-test-id=run-data-schema-item] [data-test-id=run-data-schema-item]').should('have.length', 20);
 		})
-	})
+	});
+
+	it('can link and unlink run selectors between input and output', () => {
+		cy.fixture('Test_workflow_5.json').then((data) => {
+			cy.get('body').paste(JSON.stringify(data));
+		});
+		workflowPage.actions.zoomToFit();
+		workflowPage.actions.executeWorkflow();
+		workflowPage.actions.openNode('Set3');
+
+		ndv.getters.inputRunSelector()
+			.should('exist')
```

Comment:
Is this necessary? I think next `find` should fail if the element is not there.

Reply:
if I understood correctly, this is like a wait until the element exists..